Integrations/integration-Cylance_Protect.yml

```diff
@@ -122,23 +283,23 @@ script:
     }
 
     var urlDictionary = {
-        'cp-get-list': 'globallist/get',
```

Comment:
you can't do this for backward compatibility. you should keep old commands, mark as deprecated and keep their mapping and handling